docker-compose.dev.yml

```
@@ -8,7 +8,7 @@
 # To go back to running the published image:
 #  rm docker-compose.override.yml
 
-version: '3.4'
+version: "3.4"
```

@jhonsfran1165 Could You remove this?
Main docker-compose still uses single quotes, we should keep this consistent
```suggestion
version: '3.4'
```